### PR TITLE
Minor changes to GMTMuons Dataformat to include physical variables

### PR DESCRIPTION
**Pull request by mcepeda**
Following the discussion between HLT ( @trtomei ) and @jheikkil, this PR adds physical values for Pt,Eta,Phi,D0,Z0 to the GMT Muons (SA and TK) with changes to their dataformats.

In the L1TNtuples we will keep the values of hwPt and the phPt (etc). 

I have checked this runs, and that the results seems sensible over 100 events, but I have not done a through validation. I will have little time to do so until the 8th of August. @zhenbinwu has looked at the additions to the dataformats and is OK with them.

Please do not merge this until Jaana says so :).

### CODE REVIEW

File: DataFormats/L1TMuonPhase2/interface/SAMuon.h [hunk 1]
```diff
@@ -31,6 +32,14 @@ namespace l1t {
     const uint hwBeta() const { return hwBeta_; }
     void setBeta(uint beta) { hwBeta_ = beta; }
 
+    // For HLT
+    const double phZ0() const { return Phase2L1GMT::LSBSAz0*hwZ0_;}
```
Comment (zhenbinwu) [inline DataFormats/L1TMuonPhase2/interface/SAMuon.h]: For consistency, can you call hwZ0() and hwD0() here? Thanks

File: DataFormats/L1TMuonPhase2/interface/TrackerMuon.h [hunk 2]
```diff
@@ -43,6 +44,14 @@ namespace l1t {
     void setMuonRef(const edm::Ref<l1t::RegionalMuonCandBxCollection>& p) { muRef_ = p; }
     void setHwIsoSum(int isoSum) { hwIsoSum_ = isoSum; }
     void setHwIsoSumAp(int isoSum) { hwIsoSumAp_ = isoSum; }
+ 
+    // For HLT
+    const double phZ0() const { return Phase2L1GMT::LSBGTz0*hwZ0_;}
```
Comment (zhenbinwu) [inline DataFormats/L1TMuonPhase2/interface/TrackerMuon.h]: Use hwZ0() and hwD0().

**Review by zhenbinwu — COMMENTED** [review]
Minor comments. I am OK with this PR.

File: DataFormats/L1TMuonPhase2/interface/TrackerMuon.h (hunk 2 above)
Comment (jheikkil) [reply DataFormats/L1TMuonPhase2/interface/TrackerMuon.h]: Same for hwCharge? @zhenbinwu

Comment (zhenbinwu) [reply DataFormats/L1TMuonPhase2/interface/TrackerMuon.h]: yes, please.

**Comment by epalencia** [comment]
Tagged as **l1t-phase2-v3.4.39**.

**Pull request merged**